### PR TITLE
T1558

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/1558-add-default-content-to-newscaster

### CODE REVIEW

File: templates/supported/newscaster/butter/template.js [hunk 1]
```diff
@@ -13,17 +13,75 @@
...
-            text: "This is a test.",
-            target: "Area1"
+        var trackOne = media.addTrack( "Track1" ),
+            trackTwo = media.addTrack(),
+            trackThree = media.addTrack();
+
+        trackOne.addTrackEvent({
+          "type": "text",
+          "popcornOptions": {
+            "start": "0",
```
Comment (humphd) [inline templates/supported/newscaster/butter/template.js]: no double-quotes around "0", should be 0

File: templates/supported/newscaster/butter/template.js [hunk 2]
```diff
@@ -13,17 +13,75 @@
...
+          "popcornOptions": {
+            "start": 7.477,
+            "end": 9.537,
+            "target": "video-overlay",
+            "text": "I am a Robot!",
+            "type": "thought",
+            "triangle": "top left",
+            "flip": false,
+            "classes": "none",
+            "order": "1",
```
Comment (humphd) [inline templates/supported/newscaster/butter/template.js]: Should be number not string?

File: templates/supported/newscaster/butter/template.js [hunk 3]
```diff
@@ -13,17 +13,75 @@
...
+            "top": "200",
```
Comment (humphd) [inline templates/supported/newscaster/butter/template.js]: Same with this three, should be numbers I think.

File: templates/supported/newscaster/butter/template.js [hunk 4]
```diff
@@ -13,17 +13,75 @@
...
+        
+        trackTwo.addTrackEvent({
+          "type": "photo",
+          "popcornOptions": {
+            "start": 3.988,
+            "end": 6.872,
+            "target": "video-overlay",
+            "src": "http://www.mozilla.org/img/covehead/firefox/brand-toolkit/identity-logo-firefox.png",
+            "href": "http://www.mozilla.org/",
+            "width": "150",
```
Comment (humphd) [inline templates/supported/newscaster/butter/template.js]: Numbers vs. Strings for these.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/98c6a0c7adf052b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/98c6a0c7adf052b/output.txt

Total script time: 0.67 mins

Server is running on http://pr528.butyr.org/templates/test.html

**Pull request merged**